.env.production

```
@@ -56,18 +56,18 @@ REACT_APP_PATH_REGEX_ENS="/ipfs"
 #REACT_APP_SENTRY_AUTH_TOKEN='<sentry_auth_token>'
 
 # API
-#REACT_APP_API_URL_PROD_MAINNET=https://protocol-mainnet.gnosis.io/api
-#REACT_APP_API_URL_PROD_RINKEBY=https://protocol-rinkeby.gnosis.io/api
-#REACT_APP_API_URL_PROD_XDAI=https://protocol-xdai.gnosis.io/api
+#REACT_APP_API_URL_PROD_MAINNET=https://api.cow.fi/mainnet
+#REACT_APP_API_URL_PROD_RINKEBY=https://api.cow.fi/rinkeby
+#REACT_APP_API_URL_PROD_XDAI=https://api.cow.fi/xdai
```

Same thing as on Henry's PR, sorry about that https://github.com/gnosis/gp-ui/pull/931/files#r779053245

```suggestion
#REACT_APP_API_URL_PROD_MAINNET=https://api.cow.fi/mainnet/api
#REACT_APP_API_URL_PROD_RINKEBY=https://api.cow.fi/rinkeby/api
#REACT_APP_API_URL_PROD_XDAI=https://api.cow.fi/xdai/api
```